viewer/package.json

```diff
@@ -41,13 +41,14 @@
     "react-map-gl": "^5.1.3",
     "react-map-gl-draw": "^0.16.0",
     "react-map-gl-geocoder": "^2.2.0",
-    "react-scripts": "^3.3.0",
+    "react-scripts": "5.0.1",
     "react-use": "^14.1.1",
     "react-with-styles-interface-aphrodite": "^6.0.1",
     "reactstrap": "^8.1.1",
     "reaviz": "^8.1.0",
     "serialize-javascript": ">=2.1.1",
     "styled-components": "^4.4.0",
+    "url": "^0.11.4",
```

Comment:
👋🏼 hi new dependency

Reply:
agree!! this one enters my mind often... hard to imagine now that there was a time when react router did not support hooks yet so we picked this :face_with_spiral_eyes:

Reply:
it has served as well for many years 🦾